### ISSUE TITLE
JOSS: add "Used by..." section

### ISSUE THREAD

**Issue by egpbos** [dianna-ai/dianna#140]
JOSS asks us to: "Mention (if applicable) a representative set of past or ongoing research projects using the software and recent scholarly publications enabled by it."

I think we can also add blog posts to this.

**Comment by cwmeijer** [comment]
Blocked by #229

**Comment by elboyran** [comment]
@cwmeijer , @gcroci2, can we refer to the Turkish drama project? 

@cwmeijer , @egpbos, can we refer to the future use by the Spoken language project?

Relevant info from the criteria: 
•  Whether the software has already been cited in academic papers. 
•  Whether the software is sufficiently useful that it is likely to be cited by other researchers working in this domain.

What JOSS wants in a submitted paper.

•  A list of key references, including to other software addressing related needs. Note that the references should include full names of venues, e.g., journals and conferences, not abbreviations only understood in the context of a specific discipline. 
•  Mention (if applicable) a representative set of past or ongoing research projects using the software and recent scholarly publications enabled by it.

**Comment by cwmeijer** [comment]
@elboyran, we can reference the TD project. They are still writing a blog post. I haven't seen any draft yet. In 2 weeks we'll have our last meeting to wrap up. Hopefully they'll have a blog post ready that we could cite. Is there any other specific information that we need from them for the JOSS paper? A link to the blog post is probably enough, right?

For Spoken Language I'm not sure how we could refer to it, as DIANNA wasn't used for the project yet, and the project is already officially over.

**Comment by gcroci2** [comment]
I think that a link to a TD project blog post would be the best, but also just mentioning their research project would be okay, right?(Mention a representative set of [...] **ongoing research projects** using the software [...])

In the latter case, do we have a TD project description? I asked for something similar also to Peter this morning, but maybe we already have it.

**Comment by elboyran** [comment]
@cwmeijer, JOSS criteria say mention "past or ongoing research projects using the software and recent scholarly publications enabled by it.". Blogs are not scholarly, but still something. 
If there's nothing it's trickier, we could point to a project page and state our approved future work for/related to it.
